wire-ios-sync-engine/Source/Synchronization/Strategies/CallingRequestStrategy.swift

```diff
@@ -81,13 +80,15 @@ public final class CallingRequestStrategy: AbstractRequestStrategy, ZMSingleRequ
         let selfUser = ZMUser.selfUser(in: managedObjectContext)
 
         if let clientId = selfUser.selfClient()?.remoteIdentifier {
-            self.callCenter = WireCallCenterV3Factory.callCenter(
-                withUserId: selfUser.avsIdentifier,
-                clientId: clientId,
-                uiMOC: managedObjectContext.zm_userInterface,
-                flowManager: flowManager,
-                transport: self
-            )
+            managedObjectContext.zm_userInterface.performAndWait {
```

Comment:
to prevent core data threading violations

Reply:
question: and this is called from sync context or swift concurrency pool?